src/mgmt/deploy-director-vm.sh

```diff
@@ -166,6 +168,10 @@ EOFPW
   done
 
   echo "GATEWAY=${Gateway}" >> /etc/sysconfig/network
+  echo "MTU=${eth0}" >> /etc/sysconfig/network-scripts/ifcfg-eth0
```

Comment:
Can we change the mtu variable names for the interfaces to something like eth0_mtu?

Reply:
will change it to eth0_mtu